test/e2e/package.json

```diff
@@ -67,6 +69,7 @@
 		"testarmada-magellan": "11.0.10",
 		"testarmada-magellan-local-executor": "^2.0.3",
 		"testarmada-magellan-mocha-plugin": "github:Automattic/magellan-mocha-plugin#b731cdf6f27a6fa6e45ddcba0b8741370b2df523",
+		"typescript": "^4.1.3",
```

Comment:
In our repo only the root `package.json` depends on `typescript`, no need to add it here.